backend/tracim_backend/applications/content_todo/models_in_context.py

```diff
@@ -0,0 +1,105 @@
+from sqlalchemy.orm import Session
+
+from tracim_backend.app_models.contents import content_type_list
+from tracim_backend.applications.content_todo.models import Todo
+from tracim_backend.config import CFG
+from tracim_backend.models.auth import User
+
+
+class TodoInContext(object):
```

Comment:
After the merge, we should focus on using Todo just like Content. So a TodoInContext should inherit from a Content